### DISCUSSION TITLE
Question on seed values for PBN file generation

### DISCUSSION THREAD

**Discussion by Rick-Wilson** [ADavidBailey/Practice-Bidding-Scenarios#41] [General]
Presently, the PBN files are generated using makePBN.cmd, which has lines like this:

```batchfile
P:\dealer P:\dlr\Rule_of_2-W.dlr -s=866932216 >P:\pbn\Rule_of_2-W.pbn
P:\dealer P:\dlr\Non_Leaping_Michaels_After_2-Bid-E.dlr -s=1940732167 >P:\pbn\Non_Leaping_Michaels_After_2-Bid-E.pbn
P:\dealer P:\dlr\Opps_Double_1_NT-S.dlr -s=669639072 >P:\pbn\Opps_Double_1_NT-S.pbn
``` 

The seed value is different for each scenario, but constant for each run (i.e., a given scenario will always use the same seed value, so the resulting PBN files will always contain the same hands).

I'm wondering if this is any better than just using the same seed value for each scenario.  If we can do that, it would simplify migrating this build script to be loop-based, or makefile-based.  

@ThorvaldAagaard, @ADavidBailey said it was your suggestion to use different seed values.  I wanted to check and see if that is really what you recommended, and if it really matters in this context.

**Comment by ThorvaldAagaard** [comment]
You can use the same seed for all if you don't mind the same deal pops up in different scenarios

The seed just orders decks in the same sequence, and we check if the deal match our specifications, so I recommend different string point, but on the other side a scenario should be reproducible until all criteria's has been checked.

So assuming you trust the criteria, you could just drop the seed, or you can use a fixed seed, whatever best suit your goal

**Reply by ADavidBailey** [reply]
:) I'm not sure that my 'criteria' will ever be 'set'.
